github/src/main/kotlin/gropius/sync/github/Outgoing.kt

```diff
@@ -312,6 +337,26 @@ class Outgoing(
         return handleFinalReopenCloseBlock(finalBlock, relevantTimeline, imsProjectConfig, issueInfo)
     }
 
+    /**
+     * Mutate the comments of an issue upto GitHub
+     * @param imsProjectConfig active config
+     * @param issueInfo info of the issue containing the timeline item
+     * @param timeline TimelineItems for this issue
+     * @return List of functions that contain the actual mutation executors
+     */
+    private suspend fun pushComments(
+        imsProjectConfig: IMSProjectConfig, issueInfo: IssueInfo, timeline: List<TimelineItem>
+    ): List<suspend () -> Unit> {
+        val collectedMutations = mutableListOf<suspend () -> Unit>()
+        val relevantTimeline = timeline.mapNotNull { it as? IssueComment }
+            .filter { timelineEventInfoRepository.findByNeo4jId(it.rawId!!) == null }.forEach {
```

Comment:
wouldn't it be possible to replace the foreach with a map and not use the temp mutable list?

Reply:
there may be more than one element added

Reply:
then flatmap

Reply:
d43c14e49da24e0043b1b985422fbf6633db585b